config/2.0.0/sample_config_2.0.0.json

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
The diff of this against `config/0.0.1/sample_config_0.0.1.json` might be pretty helpful to see what has fundamentally changes in the configuration file schema (similar to the diff in `tests/unit/data/test_config.json`).

Reply:
Something I was thinking about since posting the PR:

I've bumped the schema version number from 0.0.1 to 2.0.0. Largely that's because I've been referring to the updated schema as v2. But... that's skipping 1.0.0. Maybe I should use 1.0.0? (I kind of like that now I say it out loud, because it implies the cleaned up schema is the first non-beta version) Thoughts?